### ISSUE TITLE
NVDA in the windows mail and people apps doesn't read text when navigating

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4772]
**Reported by fatih on 2015-01-03 22:04**
hi,
When I press left, right, up and down arrows in the windwos mail and people apps, it doesn't read the text.
Thanks.

**Comment by leonardder** [comment]
@michaeldcurran: as you were the last who commented in this ticket, you might know the current status of this?

**Comment by michaelDCurran** [comment]
These apps have been totally redesigned since this issue was open. Closing as worksforme.
